CONTRIBUTING.rst

```diff
@@ -102,22 +102,29 @@ Make the necessary changes to prepare the new release:
     2. Update the release notes in ``HISTORY.rst``
 
 Commit the changes, open a Pull Request on GitHub and request approval from another committer.
-Once your PR has been merged, you can create the release tag and then push the artifacts to PyPI with twine::
+Once your PR has been merged, a GitHub action will automatically create the release tag and push the artifacts to PyPI.
+
+After a couple minutes, check for the new release's appearance at https://pypi.org/project/cumulusci/
+
+Next, head to the Release object that was autocreated in the GitHub repository, edit it, paste in the changelog notes and hit publish.
+
+You can then create a pull request to update the `Homebrew Tap`_ by running this locally (note, it's important to do this as soon as possible after the release is published on PyPI, because the former is the source for when CumulusCI checks to see if a new version is available)::
```

Comment:
```suggestion
You can then create a pull request to update the `Homebrew Tap`_ by running this locally (note, it's important to do this as soon as possible after the release is published on PyPI, because PyPI is the source CumulusCI checks to see if a new version is available)::
```